README.md

```diff
@@ -18,9 +18,43 @@
 ## Usage
 
 1. Choose your flavour.
-2. Copy the contents of _flavour_.cava into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
+2. Copy the contents of _flavour_.cava into your Cava config file, as specified in [Cava's README](https://github.com/karlstav/cava#configuration). Typically, the config file is located at: `~/.config/cava/`. Ensure you replace the existing gradient settings.
+
+> [!WARNING]
+> Running any of the commands below may overwrite your existing cava configuration file.
+
+  - Latte
+      ```sh
+      wget -O ~/.config/cava/config https://github.com/catppuccin/cava/raw/main/latte.cava
+      ```
+
+  - Frappé
+      ```sh
+      wget -O ~/.config/cava/config https://github.com/catppuccin/cava/raw/main/frappe.cava
+      ```
+
+  - Macchiato
+      ```sh
+      wget -O ~/.config/cava/config https://github.com/catppuccin/cava/raw/main/macchiato.cava
+      ```
+
+  - Mocha
+      ```sh
+      wget -O ~/.config/cava/config https://github.com/catppuccin/cava/raw/main/mocha.cava
+      ```
+
 3. Reload cava if it was already playing.
```

Comment:
According to [the Cava docs](https://github.com/karlstav/cava?tab=readme-ov-file#configuration), you can reload Cava without restarting by sending it `SIGUSR1` :
```sh
$ pkill -USR1 cava
```
Is it worth noting this?